sklearn/utils/validation.py

```diff
@@ -18,14 +18,13 @@
 
 from numpy.core.numeric import ComplexWarning
 
-from ..externals import six
-from ..utils.fixes import signature
+from . import Memory, joblib_version
+from .fixes import signature
 from .. import get_config as _get_config
+from ..externals import six
 from ..exceptions import NonBLASDotWarning
 from ..exceptions import NotFittedError
 from ..exceptions import DataConversionWarning
-from ..utils._joblib import Memory
-from ..utils._joblib import __version__ as joblib_version
```

Comment:
Also I'm not so keen on this, we can just use,
```py
from sklearn.utils import _joblib

_joblib.__version__
```
I think, without exposing another public attribute in utils..